static/src/javascripts/projects/common/modules/navigation/newSticky.js

```diff
@@ -107,48 +113,36 @@ define([
     };
 
     var render = function (els, state) {
-        // Reset so we have a clean slate
-        els.$header.css({
-            'transition': '',
-            'margin-top': ''
-        });
-        els.$adBanner.css({
-            'position': '',
-            'top': '',
-            'max-height': '',
-            'overflow': '',
-            'transition': ''
-        });
-
-        // Set
         // TODO: move into stylesheets when productionised
         els.$document.addClass('new-sticky-ad');
-        els.$header.css({ 'margin-top': state.adHeight });
-        els.$adBanner.css({ 'max-height': state.adHeight });
+
+        var transitionTimingFunction = 'cubic-bezier(0, 0, 0, .985)';
+        var transitionDuration = '1s';
+
+        els.$header.css({
+            'transition': state.shouldTransition
+                ? ['margin-top', transitionDuration, transitionTimingFunction].join(' ')
+                : '',
+            'margin-top': state.adHeight
+        });
 
         var pageYOffset = state.scrollCoords[1];
-        var isScrollPastStickyZone = pageYOffset > state.headerHeight;
-        if (isScrollPastStickyZone) {
-            els.$adBanner.css({
-                'position': 'absolute',
-                'top': state.headerHeight
-            });
-        } else {
-            els.$adBanner.css({ 'position': 'fixed' });
-        }
+        var userHasScrolledPastHeader = pageYOffset > state.headerHeight;
+
+        els.$adBanner.css({
+            'position': userHasScrolledPastHeader ? 'absolute' : 'fixed',
+            'top': userHasScrolledPastHeader ? state.headerHeight : '',
+            'height': state.adHeight,
+            // Stop the ad from overflowing while we transition
+            'overflow': state.shouldTransition ? 'hidden' : '',
+            'transition': state.shouldTransition
+                ? ['height', transitionDuration, transitionTimingFunction].join(' ')
+                : ''
```

Comment:
[See here](https://github.com/guardian/frontend/pull/11916/files#r52463403), and line 137?